test_cases/process_approval.js

```diff
@@ -175,32 +146,29 @@ export default (folderName, Common) => {
 
             cy.get('[data-cy="inbox_icon"]')
               .should('exist')
-              // .wait(5000)
               .click({ force: true });
 
             cy.get('.webix_accordionitem_header')
               .should('exist')
               .and('be.visible');
 
             cy.get('[data-cy="inbox-accordion-app-holder-0ac51d6c-7c95-461c-aa8b-7da00afc4f48_b17f815f-cadc-4829-afb0-a7a726bde018"]')
-              // .should('exist')
               .contains('Admin Verify')
               .scrollIntoView()
               .click();
 
-            cy.get('button[name="data"]')
-              // .should('exist')
+            cy.get('.formio-component-deny > button')
               .contains('Deny')
               .scrollIntoView()
               .click();
 
-            // cy.get('[data-cy="inbox_taskwindow_close"]')
-            //   .should('exist')
-            //   .click();
+        });
 
-            cy.get('.webix_el_box')
-              .eq(14)
-              .find('button')
+        // 4. Checking for the latest Stock
```

Comment:
I separate and check stock later because It will not pass if I check in third test.